### DISCUSSION TITLE
Help deploying Nuxt landing page to Plesk.

### DISCUSSION THREAD

**Discussion by nezlicodes** [nuxt/nuxt#10337] [Questions (Nuxt 2)]
### Versions

- nuxt: v2.15.7
- node: v16.13.0

### Reproduction

Link to a minimal test case based on one of: https://github.com/nezlicodes/bdp

<details open>
<summary>Additional Details</summary>
<br>
NodeJs config in Plesk
<img width="1392" alt="Capture d’écran 2022-03-18 à 3 41 52 PM" src="https://user-images.githubusercontent.com/40661548/159024230-9c935d43-dc56-40a4-a189-3187197fe217.png">

</details>

### Steps to reproduce
Deploy site to the repo in order to deloy it.

### What is Expected?
The site should be deployed to production.

### What is actually happening?
Plesk home page is rendered

**Comment by TheAlexLichter** [comment]
Did you take a look at https://github.com/pi0/nuxt-plesk-example already? ☺️

**Reply by nezlicodes** [reply]
Thanks for the help!
Yes, I've made some changes to match the repo's instructions but I guess it takes some time before they are deployed. 🤔

I have updated:
- the **server.js** file.
- the NodeJs version.
- the Application Startup File.
